### DISCUSSION TITLE
Minus moment during the optimization

### DISCUSSION THREAD

**Discussion by shaheidula** [mdolab/dafoam#318] [General]
Dear Prof. Ping He,
I tried to run a shape optimization for the wind turbine by modifying the previous runScript to the current version and got the minus moment during the optimization. Could you please check? please find the attached file below. 
![11](https://user-images.githubusercontent.com/66427363/173753218-d6d059f7-14c3-4805-924a-4c0a02bf131c.PNG)

**Comment by friedenhe** [comment]
I suggest you look at the flow field and figure out why the moment changed the sign during the optimization. The optimization worked fine, but the results seem to be not quite physical. It should be related to the flow, instead of optimization.

**Reply by shaheidula** [reply]
Hello Prof. Ping, 
Thanks a lot.
